.gitignore

```diff
@@ -40,3 +40,6 @@ pkg
 #
 # For vim:
 #*.swp
+
+# deps should be good enough in Gemfile and gemspec that this isn't needed
+Gemfile.lock
```

Comment:
Totally fine with that. I've been avoiding gemfile.lock because for some reason at some point the community did it that way. I'll add it in here.